go/src/koding/klient/machine/machinegroup/syncs/syncs.go

```diff
@@ -141,7 +124,23 @@ func (s *Syncs) worker() {
 
 // Add starts synchronization between remote and local directories. It creates
 // all necessary cache files if they are not present.
-func (s *Syncs) Add(mountID mount.ID, m mount.Mount, dynClient client.DynamicClientFunc) error {
+func (s *Syncs) Add(mountID mount.ID, m mount.Mount,
+	nb notify.Builder, sb msync.Builder,
+	dynSSH msync.DynamicSSHFunc, dynClient client.DynamicClientFunc) error {
```

Comment:
`AddOptions`? Then you could refactor this further and add `(*AddOptions).Valid() error`.